tidy3d/plugins/design/result.py

```diff
@@ -61,23 +62,24 @@ class Result(Tidy3dBaseModel):
         description="Source code for the function evaluated in the parameter sweep.",
     )
 
-    task_ids: Tuple[Tuple[str, ...], ...] = pd.Field(
+    task_ids: Tuple[str, ...] = pd.Field(
         None,
         title="Task IDs",
         description="Task IDs for the simulation run in each data point. Only available if "
         "the parameter sweep function is split into pre and post processing and run with "
-        "'Design.run_batch()', otherwise is ``None``. "
-        "To access all of the data, see ``batch_data``.",
+        "'Design.run_batch()', otherwise is ``None``. ",
     )
 
-    batch_data: BatchData = pd.Field(
+    aux_values: Tuple[Any, ...] = pd.Field(
         None,
-        title="Batch Data",
-        description=":class:`BatchData` object storing all of the data for the simulations "
-        " used in this ``Result``. Can be iterated through like a dictionary with "
-        "``for task_name, sim_data in batch_data.items()``. Only available if "
-        "the parameter sweep function is split into pre and post processing and run with "
-        "'Design.run_batch()', otherwise is ``None``.",
+        title="Auxiliary values output from the user function",
+        description="The auxiliary return values from the design problem function. These weren't used to inform the optimizer, if one was used.",
+    )
+
+    opt_output: Any = pd.Field(
+        None,
+        title="Plots produced through design method",
+        description="Plot objects captured over the course of the design method run.",
```

Comment:
using matplotlib? how do I plot these? some more description could be useful here. Also, I'm not sure if this will be serializable / uploadable (for future)